test/uk/gov/hmrc/helptosave/repo/EligibilityStatsStoreSpec.scala

```diff
@@ -16,14 +16,22 @@
 
 package uk.gov.hmrc.helptosave.repo
 
+import org.scalatest.BeforeAndAfterEach
 import play.api.libs.json.Json
-import play.modules.reactivemongo.ReactiveMongoComponent
 import uk.gov.hmrc.helptosave.repo.MongoEligibilityStatsStore.EligibilityStats
+import uk.gov.hmrc.helptosave.repo.MongoEnrolmentStore.EnrolmentData
 import uk.gov.hmrc.helptosave.utils.TestSupport
+import uk.gov.hmrc.mongo.MongoComponent
+import uk.gov.hmrc.mongo.test.MongoSupport
 
-class EligibilityStatsStoreSpec extends TestSupport with MongoSupport {
+class EligibilityStatsStoreSpec extends TestSupport with MongoSupport with BeforeAndAfterEach {
 
-  def newEligibilityStatsMongoStore(reactiveMongoComponent: ReactiveMongoComponent) = new MongoEligibilityStatsStore(reactiveMongoComponent, mockMetrics)
+  def newEligibilityStatsMongoStore(mongoComponent: MongoComponent) = new MongoEligibilityStatsStore(mongoComponent, mockMetrics)
+  val repository = newEligibilityStatsMongoStore(mongoComponent)
+
+  override def beforeEach(): Unit = {
+    await(repository.collection.drop().toFuture())
```

Comment:
instead of this, you can directly use dropDatabase() from MongoSupport, which does the same thing

Reply:
ok